src/wasm.h

```diff
@@ -1469,11 +1469,12 @@ class Table {
   // Currently the wasm object always 'has' one Table. It 'exists' if it has been defined or imported.
   // The table can exist but be empty and have no defined initial or max size.
   bool exists;
+  bool isImported;
```

Comment:
`imported` seems more consistent with the nearby names like `exists`. If we had `isImported`, we should have had `doesExist`.